root/components/LinkSearchableEditType.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * @flow strict
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import {URL} from 'url';
```

Comment:
You can use `URL` in Node and the browser without importing (but importing it won't work in-browser anymore): https://github.com/metabrainz/musicbrainz-server/commit/b27261bc39152692673bf1eebecdfe0730b980bb